providers/summary/scancode.js

```diff
@@ -58,79 +57,90 @@ class ScanCodeSummarizer {
     if (releaseDate) result.described = { releaseDate: releaseDate.trim() }
   }
 
-  _summarizeDiscoveredLicenseInfo(files) {
-    const copyrightHolders = new Set()
+  _summarizeFacetInfo(facet, facetFiles) {
+    if (!facetFiles || facetFiles.length === 0) return null
+    const attributions = new Set()
     const licenseExpressions = new Set()
     let unknownParties = 0
     let unknownLicenses = 0
-    for (let file of files) {
-      this._addArrayToSet(file.licenses, licenseExpressions, license => license.spdx_license_key)
-      const asserted = get(file, 'packages[0].asserted_licenses')
-      if (!asserted) {
-        if (!file.licenses || file.licenses.length === 0) unknownLicenses++
-        const hasHolders = this._normalizeCopyrights(file.copyrights, copyrightHolders)
-        !hasHolders && unknownParties++
+    for (let file of facetFiles) {
+      file.license ? licenseExpressions.add(file.license) : unknownLicenses++
+      file.attributions ? this._addArrayToSet(file.attributions, attributions) : unknownParties++
+      if (facet !== 'core') {
+        // tag the file with the current facet if not core
+        file.facets = file.facets || []
+        file.facets.push(facet)
```

Comment:
What is being added to the file.facets array?

Reply:
the list of facet names that apply to the individual file